tests/ui/consts/const-eval/raw-pointer-ub.stderr

```diff
@@ -11,28 +11,26 @@ LL |     *ptr = 0;
    |     ^^^^^^^^ accessing memory based on pointer with alignment 1, but alignment 4 is required
 
 error[E0080]: evaluation of constant value failed
-  --> $SRC_DIR/core/src/intrinsics/mod.rs:LL:COL
+  --> $DIR/raw-pointer-ub.rs:19:5
    |
-   = note: accessing memory with alignment 1, but alignment 4 is required
+LL |     y.copy_to_nonoverlapping(&mut z, 1);
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ accessing memory with alignment 1, but alignment 4 is required
    |
-note: inside `copy_nonoverlapping::<u32>`
-  --> $SRC_DIR/core/src/intrinsics/mod.rs:LL:COL
 note: inside `std::ptr::const_ptr::<impl *const u32>::copy_to_nonoverlapping`
```

Comment:
Why does this still start with "inside" and not "the evaluated program failed inside"?

Reply:
"the evaluated program" gets only added (and gets a span label) *only* at the bottom of the stack.

Reply:
I find that confusing. What is the point of that?

Reply:
Having the same *long* message over and over again on deeply nested cases makes it easier to miss the useful information that the user needs. I don't see the benefit of having "the evaluated program failed inside `xxx`" over and over again, when the actual "failure" is on the inner-most frame (where the panic happened), and not any of the intermediary ones.

Reply:
I believe the outer-most is the most important (hence why I reversed the order) and the inner-most frame is the second most important, so I feel like it's worth it to highlight it in some way. Having said that, I've reverted the note message to be "inside `name`", but left the additional span label. Let me know if you want me to remove it as well.

Reply:
The innermost frame already stands out by being at the bottom of the list, and I agree with using the span label only there.

Can you show and updated example for what the diagnostic looks like now? (Or is there a stderr file that serves as a good example, with at least 3 frames in the stacktrace?)